### PR TITLE
Remove unused imports in Python code and unused dependency

### PR DESCRIPTION
**Pull request by johannaengland**
To add F4 rules to ruff. 

Tagging as discussion to invite discussion if/which of these rules we want. 

Reference: https://docs.astral.sh/ruff/rules/#pyflakes-f

Also removes a dependency that had become unused.

### CODE REVIEW

**Review by lunkwill42 — COMMENTED** [review]
I think this is a worthwhile linter rule.  Unused imports tend to clutter the codebase as it grows unless we have something to remind us to prune them.  @hmpf ?

File: src/argus_htmx/tests.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-from django.test import TestCase
+from django.test import TestCase  # noqa: F401 - unused-import
```
Comment (hmpf) [inline src/argus_htmx/tests.py:1]: Better to remove the file, it's autogenerated.

File: src/argus_htmx/models.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
-from django.db import models
+from django.db import models  # noqa: F401 - unused-import
```
Comment (hmpf) [inline src/argus_htmx/models.py:1]: This file will be used by the userprefs-branch and should stay even if it was autogenerated.

File: src/argus_htmx/themes/views.py [hunk 3]
```diff
@@ -1,13 +1,7 @@
-import importlib_resources
```
Comment (hmpf) [inline src/argus_htmx/themes/views.py:1]: This has a dependency that can be removed, see "pyproject.toml".

**Review by hmpf — CHANGES_REQUESTED** [review]
This renders a dependency redundant, see comment.

**Review by hmpf — APPROVED** [review]

**Pull request merged**